docs/source/app_development/create.rst

```diff
@@ -4,137 +4,203 @@
 Create your AiiDAlab app
 ************************
 
+There are two approaches to develop a new AiiDAlab app:
+
+.. panels::
+   :body: bg-light
+   :footer: bg-light border-0
+
+   :fa:`bolt` App cutter
+
+   The easiest and recommended approach is to create a new app with the :ref:`AiiDAlab app cookiecutter <develop-apps:create-app:variant-a_cookiecutter>`.
+
+   ---
+
+   :fa:`wrench` Step-by-step from scratch
+
+   Of course, one can also create an app :ref:`step by step from scratch <develop-apps:create-app:variant-b_from-scratch>`.
+
+
 .. _develop-apps:create-app:variant-a_cookiecutter:
 
-Variant A: Quickstart using the AiiDAlab app cookiecutter
-=========================================================
+Create app with cookiecutter
+============================
+
+Open the terminal from the AiiDAlab app start page by clicking on the "Terminal" :fa:`terminal` icon on the start page.
```

Comment:
```suggestion
Open the terminal from the AiiDAlab app start page by clicking on the "Terminal" (:fa:`terminal`) icon on the start page.
```

Just to better separate the icon from the other text. Otherwise, it looks like there are no spaces.